checkpoint_quantum_firewall/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# CHANGELOG - checkpoint_quantum_firewall
+
+<!-- checkpoint_quantum_firewall release notes start -->
+
+## 1.0.0 / 2024-04-25
+
+***Added***:
+
+* Adds Check Point Next Generation Firewall integration.
```

Comment:
Could you revert [this](https://github.com/DataDog/integrations-core/pull/16522#discussion_r1537931375)? The changelog should be empty for agent integrations, it will be generated during the release.

Reply:
Done